src/js/common/components/form-elements/GrowableTable.jsx

```diff
@@ -16,7 +16,7 @@ class GrowableTable extends React.Component {
     this.handleEdit = this.handleEdit.bind(this);
     this.handleSave = this.handleSave.bind(this);
     this.handleRemove = this.handleRemove.bind(this);
-    this.state = {};
+    this.state = { edited: false };
```

Comment:
It's a little confusing to me what all the state changes might end up looking like. I wonder if it would be useful to add a comment with what the state object might look like after a few interactions?